.github/workflows/windows.yml

```
@@ -0,0 +1,59 @@
+name: Windows CI
+
+on:
+  push:
+    branches: [ "main" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)